site/go.mod

```diff
@@ -3,3 +3,5 @@ module github.com/agones/agones/site
 go 1.20
 
 require gopkg.in/yaml.v2 v2.2.8
```

Comment:
Shouldn't there also be a change in the config.toml and a hugo.toml? - looking at https://www.docsy.dev/docs/updating/convert-site-to-module/

I feel like this is just using our existing docsy theme in the `docsy` folder?

Reply:
Yes, I've followed the same instructions from https://www.docsy.dev/docs/updating/convert-site-to-module/
and received this error -> https://gist.github.com/Kalaiselvi84/69383165776fc8d7cd450cbc770f6853

Reply:
> Shouldn't there also be a change in the config.toml and a hugo.toml? - looking at https://www.docsy.dev/docs/updating/convert-site-to-module/
> 
> I feel like this is just using our existing docsy theme in the `docsy` folder?

Yes, but you would need to migrate first:

So you have to do https://www.docsy.dev/docs/updating/convert-site-to-module/